apps/openmw/mwrender/actoranimation.cpp

```diff
@@ -136,9 +133,9 @@ bool ActorAnimation::updateCarriedLeftVisible(const int weaptype) const
         MWMechanics::CreatureStats &stats = cls.getCreatureStats(mPtr);
         if (cls.hasInventoryStore(mPtr) && weaptype != ESM::Weapon::Spell)
         {
-            SceneUtil::FindByNameVisitor findVisitor ("Bip01 AttachShield");
-            mObjectRoot->accept(findVisitor);
-            if (findVisitor.mFoundNode || (mPtr == MWMechanics::getPlayer() && mPtr.isInCell() && MWBase::Environment::get().getWorld()->isFirstPerson()))
+            osg::Group* foundNode = getBoneByName ("Bip01 AttachShield");
```

Comment:
This line doesn't compile:
```
/home/elsid/dev/openmw/apps/openmw/mwrender/actoranimation.cpp: In member function ‘virtual bool MWRender::ActorAnimation::updateCarriedLeftVisible(int) const’:
/home/elsid/dev/openmw/apps/openmw/mwrender/actoranimation.cpp:136:51: error: passing ‘const MWRender::ActorAnimation’ as ‘this’ argument discards qualifiers [-fpermissive]
  136 |             osg::Group* foundNode = getBoneByName ("Bip01 AttachShield");
      |                                     ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /home/elsid/dev/openmw/apps/openmw/mwrender/actoranimation.cpp:1:
/home/elsid/dev/openmw/apps/openmw/mwrender/actoranimation.hpp:44:21: note:   in call to ‘osg::Group* MWRender::ActorAnimation::getBoneByName(const string&)’
   44 |         osg::Group* getBoneByName(const std::string& boneName);
      |                     ^~~~~~~~~~~~~
```

Reply:
https://gitlab.com/psi29a/openmw/-/jobs/1578145134

Another reason to have basic CI on github if we plan on having contributors on github still. :/

I can't keep running interference on my side.